Makefile

```diff
@@ -1,7 +1,7 @@
 VERSION := $(shell git describe --tags --dirty=-modified --always)
 CONTROLLER_IMAGE := quay.io/skupper/controller:v2-latest
 BOOTSTRAP_IMAGE := quay.io/skupper/bootstrap:v2-latest
-CONFIG_SYNC_IMAGE := quay.io/skupper/config-sync:v2-latest
+CONFIG_SYNC_IMAGE := quay.io/skupper/kube-adaptor:v2-latest
```

Comment:
I believe that the variable must also be renamed to: `ADAPTOR_IMAGE` (based on circleci changes).

Reply:
Thanks! Well spotted! I have pushed a fix for that now.